src/App.css

```diff
@@ -1,5 +1,276 @@
-* {
+*,
+*::before,
+*::after {
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
   box-sizing: border-box;
-  margin: 0;
+}
+
+* {
   padding: 0;
+  margin: 0;
+}
+
+html {
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+  height: 100%;
+  font-family: RobotoSlab, Montserrat, sans-serif;
+}
+
+body {
+  height: 100%;
+}
+
+#root {
+  height: 100%;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.05);
+}
+
+.wrapper {
+  width: 100%;
+  position: fixed;
+  top: 0;
+}
+
+.nav-bar {
+  width: 100%;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 1rem 10%;
+  position: relative;
+  top: 0;
+  background-color: white;
+  font-family: Montserrat, RobotoSlab, sans-serif;
+  gap: 3rem;
+}
+
+.nav-bar > p {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 0.6rem;
+  color: #0290ff;
+  font-size: 2.4rem;
+  font-weight: bold;
+}
+
+.links-nav {
+  display: flex;
+  list-style: none;
+  gap: 2rem;
+}
+
+.links-nav > li > a {
+  color: inherit;
+  text-decoration: none;
+  font-family: 'Fredoka One', cursive;
+  font-size: 1.4rem;
+}
+
+.add-book-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 90%;
+  padding: 3rem 0;
+  gap: 1.5rem;
+}
+
+.add-book-section-title {
+  font-family: Montserrat, sans-serif;
+  font-size: 1.6rem;
+  font-weight: bold;
+  letter-spacing: -0.18px;
+  color: rgb(172, 172, 172);
+}
+
+.books-list {
+  display: flex;
```

Comment:
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)